client/utils/trpc/trpc-client.ts

```diff
@@ -49,10 +49,7 @@ export const trpc = createTRPCNext<AppRouter>({
               ? {
                   authorization: `Bearer ${token}`,
                   "x-device-nonce": deviceNonce || "",
-                  "x-application-id":
```

Comment:
I've removed these headers / ENV variables. The only one we need is `x-client-id`, which should be used to load the associated `Application` when needed.